AAI/AAIConnectProfile.md

```diff
@@ -803,6 +814,209 @@ following:
+end ref
+
+==Use==
+
+user -> client : User instructs client\nregarding data access
+client -> clearing : Client asks for data
+note over client, clearing  #CCCCCC
+{
+   Authorization: Bearer <Passport-Scoped OAuth Access Token>
+}
+end note
+
+clearing -> broker : Please give me details of the passport
+note over broker, clearing  #CCCCCC
+{
+  Authorization: Bearer <Passport-Scoped OAuth Access Token>
+}
+end note
+
+broker -> issuers : (optional) Fetch signed visa(s) for user
+note over broker, issuers
+[
+  <visa1>,
+  <visa2>,
+]
+end note
+broker <- issuers : (optional) Return signed visa(s) for user
+
+
+clearing <- broker : Return passport
+note over broker, clearing  #CCCCCC
+{
+  "iss": "https://broker.example.com",
+  "sub": <clientid>,
+  "jti": "071f99cc-0b52-11ec-96c2-374f836e9e79",
+  "ga4gh_passport_v1": [
+    <visa1>,
+    <visa2>,
+    ...
+  ]
+}
+end note
+
+
+note over clearing, data  #FFCCCB
+Decision is made to release data using information contained in the
+passport - and this decision is coordinated with the data system to
+facilitate that actual data release.
+end note
+
+client <- clearing : Return data
+
+
+{% endplantuml %}
+
+#### Exchange Flow
+
+The exchange flow does not ever distribute the initial *Passport-Scoped OAuth Access Token* beyond
+the client application. A token exchange operation is executed by the client, in
+exchange for a *Passport Access Token* (need precise name), a *Work Order Token* (same) - or any
+other token that may be used downstream to access resources. In this example flow, the
+ *Passport Access Token* is used as a bearer token - though noting that the potential large size
+and overly generous scope of these tokens may limit the general availability of this technique.
+
+{% plantuml %}
+
+hide footbox
+skinparam BoxPadding 10
+skinparam ParticipantPadding 20
+
+box "Researcher"  #eee
+actor       "User Agent"                as user
+participant Client                      as client
+end box
+
+box "AAI"
+participant Broker                      as broker
+collections "IdP"                       as idps
+end box
+
+box "Data Owner"
+collections "Visa Issuer(s)"            as issuers
+end box
+
+box "Data Holder"
+participant "Clearing House"            as clearing
+participant "Data"            as data
+end box
+
+==OIDC==
+
+ref over user, client, broker, idps
+OIDC flow results in the client holding a *Passport-Scoped OAuth Access Token*.
+end ref
+
+==Exchange==
+
+user -> client : User instructs client\nregarding data access
+client -> broker : Token exchange
+note over client, broker
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=urn:ietf:params:oauth:grant-type:token-exchange
+requested_token_type=TBD
+subject_token=<Passport-Scoped OAuth Access Token>
+subject_token_type=urn:ietf:params:oauth:token-type:access_token
+end note
+
+broker -> issuers : (optional) Fetch signed visa(s) for user
+note over broker, issuers
```

Comment:
The same as above, this note shows content of the synchronous message **Return signed visa(s) for user** so it should go **after** it, not before it.